com.ibm.team.build.hjplugin/src/main/java/com/ibm/team/build/internal/hjplugin/RTCScm.java

```diff
@@ -1103,8 +1112,13 @@ public boolean checkout(AbstractBuild<?, ?> build, Launcher arg1,
     		throw new AbortException(Messages.RTCScm_checkout_failure4(e.getMessage()));
     	}
 
-
-		return true;
+	}
+	
+	@Override
+	public SCMRevisionState calcRevisionsFromBuild(Run<?, ?> build,
+			FilePath workspace, Launcher launcher, TaskListener listener)
+			throws IOException, InterruptedException {
+		return SCMRevisionState.NONE;
```

Comment:
BTW this smells like a bug in the plugin which would make polling not work properly. It is supposed to return some representation of the revision from the previous build, so `checkout` can get that `baseline` and properly determine the changelog for a new build.